app/javascript/elm/src/Main.elm

```diff
@@ -897,6 +889,14 @@ snippetGoogleTagManager =
         ]
 
 
+view : Model -> Html Msg
+view model =
+    div [ id "elm-app", class (Theme.toString model.theme) ]
+        [ viewNav model.navLogo model.isNavExpanded model.sensors model.selectedSensorId model.page
+        , viewMain model
+        ]
```

Comment:
it appears to me that this change (moving the `view` function down) wasn't necessary